README.md

```diff
@@ -30,8 +30,6 @@ are looking forward to your participation.
 **NOTE**: the scheduler is built based on [kube-batch](https://github.com/kubernetes-sigs/kube-batch);
 refer to [#241](https://github.com/volcano-sh/volcano/issues/241) and [#288](https://github.com/volcano-sh/volcano/pull/288) for more detail.
 
-[JobFlow](./docs/design/jobflow/jobflow.md) is a workflow engine based on volcano Job. It proposes two concepts to automate running multiple batch jobs, named JobTemplate and JobFlow, so end users can easily declare their jobs and run them using complex control primitives such as sequential or parallel execution, if-then -else statement, switch-case statement, loop execution, etc.
```

Comment:
It is not recommended to delete all introductions related to jobflow in the readme. You can keep the brief introduction and direct it to the design doc you write.

Reply:
got it, after #2383, i will modify this pr to get to suitable